tilt.md

```diff
@@ -32,6 +32,15 @@ operator-controller requires
 their own Tiltfiles, before proceeding. If you want to use Tilt, make sure you specify a unique `--port` flag to each
 `tilt up` invocation.
 
+## Install tilt-support Repo, one level up
+You must install the tilt-support repo one level up from this repo
```

Comment:
I do not think this text `one level up from this repo` is adding any extra value as the below command clearly mentions which repo to clone.